wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -92,6 +94,7 @@ public final class Constants {
                 org.dash.wallet.common.util.Constants.INSTANCE.setEXPLORE_GC_FILE_PATH("explore/explore.db");
                 SUPPORTS_PLATFORM = !is32Bit;
                 SUPPORTS_INVITES = false;
+                SUPPORTS_TXMETADATA = false;
```

Comment:
This will disable publishing tx-metadata to platform, which is an unfinished feature.